JiraPS/Public/Invoke-JiraIssueTransition.ps1

```diff
@@ -41,6 +41,9 @@ function Invoke-JiraIssueTransition {
         [String]
         $Comment,
 
+        [String]
```

Comment:
any particular reason why not to use `[TimeSpan]`?

Reply:
it's much common practice to use in JQL things like 10m, 8h, 3w and people get used to it

Reply:
Also TimeSpan is more .NET solution rather than Powershell. It should be simple to use the Module

Reply:
well...
we already have the `[TimeSpan]` behavior in the module (for the worklog to be precise)

https://github.com/AtlassianPS/JiraPS/blob/947628c55b58dfd8cc4acbdf889fb4f33cb88b39/JiraPS/Public/Add-JiraIssueWorklog.ps1#L36-L38